pages/dkp/kaptain/2.0.0/configuration/custom-domain-certs/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: layout.pug
+navigationTitle: Custom domain name and certificates
+title: Configure a custom domain name and certificates for Kaptain
+menuWeight: 70
+excerpt: Configure a custom domain name and certificates for Kaptain
+beta: false
+enterprise: false
+---
+
+Learn how to configure a custom domain name with TLS certificates trusted by your organization.
+This will permit browsers to validate the certificate for the Kaptain endpoint that users log on to to access the main Kaptain UI, the central dashboard.
```

Comment:
```suggestion
This will permit browsers to validate the certificate for the Kaptain endpoint that users log on to access the main Kaptain UI, the central dashboard.
```